service/rtc/simulcast.go

```diff
@@ -158,7 +158,8 @@ func (s *session) handleSenderBitrateChange(downRate int, lossRate int) (bool, i
 	}
 
 	s.mut.RLock()
-	sender := s.screenTrackSender
+	// For now we only support simulcast on VP8.
+	sender := s.screenTrackSenders[webrtc.MimeTypeVP8]
```

Comment:
This should be safe, considering we disable AV1 on the client when simulcast is on. Also, we may deprecate simulcast soon if we don't fix it for good.